### ISSUE TITLE
Robust code by default

### ISSUE THREAD

**Issue by szepeviktor** [chesio/bc-woocommerce-mpay24-gateway#3]
@chesio Do you plan to introduce phpstan? https://github.com/chesio/bc-woocommerce-mpay24-gateway/pull/2#issue-302055138

**Comment by chesio** [comment]
Yeah, I want to. The server I'm currently developing on has only PHP 7.3 available as PHP-CLI, so I'm running into the issue (GiacoCorsiglia/wordpress-stubs#13) you are well aware of... But I'll set up PHPStan in the project as soon as I'll get to another machine (perhaps at home).

**Comment by szepeviktor** [comment]
@chesio I had enough of that long-standing error. My pkg comes with an [autofixer](https://github.com/szepeviktor/phpstan-wordpress/blob/master/src/Composer/FixWpStubs.php): https://github.com/szepeviktor/phpstan-wordpress/blob/master/example/composer.json#L19-L22

**Comment by chesio** [comment]
Thanks, I'll check it out!

**Comment by szepeviktor** [comment]
Now https://packagist.org/packages/php-stubs/wordpress-stubs is available 🚀 

:)

**Comment by chesio** [comment]
I forgot about this issue, TBH ;-) I don't have much time to work on this plugin now, but I'll get to it eventually!
